Vagrantfile

```diff
@@ -31,6 +31,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   config.vm.provider "virtualbox" do |vb, override|
+    config.vm.synced_folder ".", "/srv/zulip", type: "rsync", rsync__args: ["--verbose", "--archive", "--delete", "-z"]
```

Comment:
Can you add a comment here explaining and/or linking to the article, docs, or whatever why this is correct?